.travis.yml

```diff
@@ -68,13 +68,10 @@ addons:
 language: cpp
 compiler: clang
 
-before_install:
-    - curl --retry 10 --retry-max-time 120 -L "https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2" | tar xfj -
```

Comment:
As discussed earlier, Travis has a concept of cached env. But caches are kept of-site, downloaded every time. Travis recommends against caches as it takes equal or longer time to download the cache as re-building the env. Saves spaces, costs time. This is the one thing I don't like about Travis. Noadays disk space is cheap, time is expensive.